WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -490,6 +496,16 @@ private void updateStatusTextAndImage(TextView txtStatus, ImageView imgStatus, P
                         statusIconResId = R.drawable.ic_gridicons_page;
                         statusColorResId = R.color.alert_red;
                         break;
+                    case UNKNOWN:
+                        // no-op
+                        break;
+                    case PUBLISHED:
+                        // no-op
+                        break;
+                    default:
+                        if (BuildConfig.DEBUG) {
+                            throw new IllegalStateException("Missing switch case.");
```

Comment:
Same comment as [this one](https://github.com/wordpress-mobile/WordPress-Android/pull/7985#discussion_r199888663)